ui/voice/@build/lexicon/moves-vn-lex.json

```diff
@@ -0,0 +1,86 @@
+  { "in": "sáu", "tok": "6", "tags": [ "rank", "move", "choice" ] },
+  { "in": "bảy", "tok": "7", "tags": [ "rank", "move", "choice" ] },
+  { "in": "tám", "tok": "8", "tags": [ "rank", "move", "choice" ] },
+
+  { "in": "tốt", "tok": "P", "tags": [ "role", "move" ] },
+  { "in": "ngựa", "tok": "N", "tags": [ "role", "move" ] },
+  { "in": "mã", "tok": "N", "tags": [ "role", "move" ] },
+  { "in": "tượng", "tok": "B", "tags": [ "role", "move" ] },
+  { "in": "tịnh", "tok": "B", "tags": [ "role", "move" ] },
+  { "in": "xe", "tok": "R", "tags": [ "role", "move" ] },
+  { "in": "hậu", "tok": "Q", "tags": [ "role", "move" ] },
+  { "in": "vua", "tok": "K", "tags": [ "role", "move" ] },
+
+  { "in": "nhập thành", "val": "castle", "tags": [ "move" ] },
+  { "in": "nhập thành gần", "val": "O-O", "tags": [ "move" ] },
+  { "in": "nhập thành cánh vua", "val": "O-O", "tags": [ "move" ] },
+  { "in": "nhập thành xa", "val": "O-O-O", "tags": [ "move" ] },
+  { "in": "nhập thành cánh hậu", "val": "O-O-O", "tags": [ "move" ] },
+  { "in": "ăn", "val": "x", "tags": [ "move" ] },
+  { "in": "phong", "val": "=", "tags": [ "move" ] },
+
+  { "in": "hoãn", "val": "takeback", "tags": [ "command" ] },
+  { "in": "khoan", "val": "takeback", "tags": [ "command" ] },
+  { "in": "hoà", "val": "draw", "tags": [ "command", "round" ] },
+  { "in": "xin hoà", "val": "draw", "tags": [ "command", "round" ] },
+  { "in": "chấp nhận", "val": "draw", "tags": [ "command", "round" ] },
+  { "in": "thua", "val": "resign", "tags": [ "command", "round" ] },
+  { "in": "xin thua", "val": "resign", "tags": [ "command", "round" ] },
+  { "in": "chơi lại", "val": "rematch", "tags": [ "command", "round" ] },
+  { "in": "ván nữa", "val": "rematch", "tags": [ "command", "round" ] },
+  { "in": "xoay", "val": "flip", "tags": [ "command" ] },
+  { "in": "đồng hồ", "val": "clock", "tags": [ "command", "round" ] },
+  { "in": "đối thủ", "val": "who", "tags": [ "command", "round" ] },
+  { "in": "tiếp theo", "val": "next", "tags": [ "command", "puzzle" ] },
+  { "in": "tiếp", "val": "next", "tags": [ "command", "puzzle" ] },
+  { "in": "tốt", "val": "upvote", "tags": [ "command", "puzzle" ] },
+  { "in": "thích", "val": "downvote", "tags": [ "command", "puzzle" ] },
+  { "in": "không thích", "val": "solve", "tags": [ "command", "puzzle" ] },
+  { "in": "giúp đỡ", "val": "help", "tags": [ "command" ] },
+  { "in": "dừng", "val": "stop", "tags": [ "command" ] },
+
+  { "in": "đúng", "val": "yes", "tags": [ "choice" ] },
+  { "in": "ok", "val": "yes", "tags": [ "choice" ] },
+  { "in": "xác nhận", "val": "yes", "tags": [ "choice" ] },
+  { "in": "không", "val": "no", "tags": [ "choice" ] },
+  { "in": "sai", "val": "no", "tags": [ "choice" ] },
+  { "in": "xoá", "val": "no", "tags": [ "choice" ] },
+  { "in": "đóng", "val": "no", "tags": [ "choice" ] },
+  { "in": "huỷ bỏ", "val": "no", "tags": [ "choice" ] },
+
+  { "in": "xanh nước biển", "val": "blue", "tags": [ "color", "choice" ] },
```

Comment:
disambiguation using colors won't work using my current implementation as it's heavily dependent on single word partial utterances.  hopefully numbers will work ok.

Reply:
perhaps we could find replacements for orange and green (are red & black one word?).  i think we can probably find something that works ok.

Reply:
fun fact, in vietnamese, green and blue are the same. Both are `xanh` in vietnamese. In order to say exactly green or blue we have to add something, Blue is sea-like xanh and green is leafy xanh. So, for blue and green, we have to have multiple words :(.

For `orange`, we can use only `cam` instead of `da cam`.

Reply:
> (are red & black one word?)

In vietnamese, one word is one sound, so, all of the colors except green, blue, and orange are one sound/word.

Reply:
sounds like we could swap out blue for red, green will get xanh, and orange will be cam?

Reply:
would red be đỏ?

Reply:
yeah, that would make sense.